### PR TITLE
 Added option to specify IS NULL / IS NOT NULL type of where clause

### LINKED ISSUE #6: Handling `IS NULL` and `IS NOT NULL`
Hi.. currently, it seems impossible to make a filter which translates to `IS NULL` and `IS NOT NULL`.

all my attempts created.. 

`WHERE (`employee`.`dept` NOT != (NULL))`
OR 
`WHERE (`employee`.`dept` NOT != ('null'))`

but SQL needs IS and IS NOT and no brackets around null.

I might be able to offer you a PR with a fix if you are willing to accept.

### PR DESCRIPTION
**Pull request by dharapvj**
Created new branch for PR with right commits...

Fixes https://github.com/gentritabazi/typeorm-simple-query-parser/issues/6

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ lt | Lesser than
 lte | Lesser than or equalTo
 in | In array
 bt | Between
+nullcheck | is / is not null
```
Comment (gentritabazi) [inline readme.md]: I would use "nch" // Null check

Comment (dharapvj) [reply readme.md]: Done

**Review by gentritabazi — APPROVED** [review]

**Comment by gentritabazi** [comment]
Like you can see "CI / Linters (pull_request) " is failing.

Please format code with `npm run code:format`.

**Comment by dharapvj** [comment]
done

**Comment by dharapvj** [comment]
anyways to get a release cut on npm?

**Comment by gentritabazi** [comment]
A new version of the package typeorm-simple-query-parser (1.0.17)

**Comment by dharapvj** [comment]
Much appreciated! 🙏

**Pull request merged**